### DISCUSSION TITLE
Stuck at this file when launching

### DISCUSSION THREAD

**Discussion by Heathen** [hlky/stable-diffusion-webui#791] [Q&A]
Tried launching several times, sometimes I get 30mb/s for about 1gb of file and then it stops or slows down to a crawl.
![image](https://user-images.githubusercontent.com/474879/188972875-e8517b13-bb53-4352-9e24-6dfc08ebe70d.png)

Any way I can get this model from the web and place it manually? It appears to be a transformer from huggingface.

Additional info:
OSError: Can't load the model for 'openai/clip-vit-large-patch14'. If you were trying to load it from 'https://huggingface.co/models', make sure you don't have a local directory with the same name. Otherwise, make sure 'openai/clip-vit-large-patch14' is the correct path to a directory containing a file named pytorch_model.bin, tf_model.h5, model.ckpt or flax_model.msgpack.

Assuming that's the file not working, it must be one of these models, but which one and where it goes, I have no idea.
https://huggingface.co/openai/clip-vit-large-patch14/tree/main

Took 57 restarts for it to download...... So I guess nevermind. :I

**Comment by ZeroCool22** [comment]
Hi can you show me your folder please?:

![Screenshot_8](https://user-images.githubusercontent.com/13344308/189008977-44d2ade6-e723-4ff2-842b-16611a3da9c8.jpg)

**Reply by Heathen** [reply]
It already managed to download, took over 50 tries and my .cache folder ended with 40+gb of waste.

**Reply by ZeroCool22** [reply]
> It already managed to download, took over 50 tries and my .cache folder ended with 40+gb of waste.

Ok, i just want to be sure how your folder looks:

![Screenshot_13](https://user-images.githubusercontent.com/13344308/189012468-aa399429-af73-40b5-811a-3bce7028c5eb.jpg)

**Reply by Heathen** [reply]
Folders are alright, just like you showed and everything is working, including LDSR. The file it was having trouble downloading was openai/clip-vit-large-patch14. One of the dependencies and I assume it resides in the miniconda environment.

Thanks for the help tho. :>
